suzieq/poller/worker/services/interfaces.py

```diff
@@ -952,6 +953,28 @@ def _common_data_cleaner(self, processed_data, _):
 
         return processed_data
 
+    def _clean_fwsm_data(self, processed_data, raw_data):
+        for entry in processed_data:
+            name = entry["ifname"]
+            if name.startswith("Vlan"):
+                with contextlib.suppress(Exception):
+                    vlan = int (name[4:])
+                    entry["vlan"] = vlan
```

Comment:
In which case do you expect an exception here? I think it'd be better to use a try ... except if you want to handle a specific situation or let the original exception propagate and be catched at the upper level. The latter would make it easier to notice if the output is different than expected and issue a fix.